### PR TITLE
Internal: Show promotion for DC [ED-13000]

### PR DESCRIPTION
**Pull request by murzin-nikita**
## PR Checklist

- [ ] The commit message follows our guidelines:  https://github.com/elementor/elementor/blob/master/.github/CONTRIBUTING.md

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Summary

This PR can be summarized in the following changelog entry:

*

## Description
An explanation of what is done in this PR

*

## Test instructions
This PR can be tested by following these steps:

*

## Quality assurance

- [ ] I have tested this code to the best of my abilities
- [ ] I have added unittests to verify the code works as intended
- [ ] Docs have been added / updated (for bug fixes / features)

Fixes #

### CODE REVIEW

File: modules/promotions/assets/scss/editor.scss [hunk 1]
```diff
@@ -0,0 +1,46 @@
+@import "../../../../assets/dev/scss/helpers/variables";
+
+.e-control-display-conditions-promotion {
+	&__wrapper {
+		display: flex;
+		justify-content: space-between;
+
+		&:hover {
+			//background-color: green;
```
Comment (wittytzipporah) [inline modules/promotions/assets/scss/editor.scss:9]: Commented out?

Comment (mykytamurzin) [reply modules/promotions/assets/scss/editor.scss]: @wittytzipporah fixed

File: includes/managers/controls.php [hunk 2]
```diff
@@ -1214,4 +1214,31 @@ private function should_clean_stack( $stack ): bool {
 
 		return true;
 	}
+
+	public function add_display_conditions_controls( Controls_Stack $controls_stack ) {
+		ob_start();
+		?>
+		<div class="e-control-display-conditions-promotion__wrapper">
+			<div class="e-control-display-conditions-promotion__desc">
```
Comment (hein-obox) [inline includes/managers/controls.php:1222]:
```suggestion
			<div class="e-control-display-conditions-promotion__description">
```

File: includes/managers/controls.php [hunk 3]
```diff
@@ -1214,4 +1214,31 @@ private function should_clean_stack( $stack ): bool {
...
+				<span class="e-control-display-conditions-promotion__text">
+					<?php echo esc_html__( 'Display Conditions', 'elementor' ); ?>
+				</span>
+				<span class="e-control-display-conditions-promotion__lock_wrapper">
```
Comment (hein-obox) [inline includes/managers/controls.php:1226]:
```suggestion
				<span class="e-control-display-conditions-promotion__lock-wrapper">
```

File: modules/promotions/assets/scss/editor.scss [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+		justify-content: space-between;
+
+		&:hover {
+			.eicon-lock.e-control-display-conditions-promotion {
+				color: var(--e-a-color-accent)
+			}
+		}
+	}
+
+	&__desc {
```
Comment (hein-obox) [inline modules/promotions/assets/scss/editor.scss:15]:
```suggestion
	&__description {
```

File: modules/promotions/assets/scss/editor.scss [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+		display: flex;
+		align-self: center;
+	}
+
+	&__text {
+		display: inline-block;
+	}
+
+	&__lock_wrapper {
```
Comment (hein-obox) [inline modules/promotions/assets/scss/editor.scss:24]:
```suggestion
	&__lock-wrapper {
```

File: includes/elements/column.php [hunk 6]
```diff
@@ -822,6 +822,8 @@ protected function register_controls() {
 			]
 		);
 
+		Plugin::$instance->controls_manager->add_display_conditions_controls( $this );
```
Comment (hein-obox) [inline includes/elements/column.php:825]: What happens with this control in the Pro version?

File: assets/dev/scss/editor/editor.scss [hunk 7]
```diff
@@ -18,6 +18,7 @@
 @import "mode-switcher";
 @import "../../../../modules/history/assets/scss/revisions";
 @import "../../../../modules/history/assets/scss/history";
+@import "../../../../modules/promotions/assets/scss/editor";
```
Comment (hein-obox) [inline assets/dev/scss/editor/editor.scss:21]: Interesting project.

File: includes/elements/column.php (hunk 6 above)
Comment (mykytamurzin) [reply includes/elements/column.php:812]: @hein-obox I have a PR to remove promotion control in PRO
https://github.com/elementor/elementor-pro/pull/3395

**Review by hein-obox — APPROVED** [review]

**Pull request merged**